core/src/main/java/com/radixdlt/RadixNode.java

```diff
@@ -154,6 +155,10 @@ public static RadixNode run(
     final var engineStateApiServer = injector.getInstance(EngineStateApiServer.class);
     engineStateApiServer.start();
 
+    // Start the Mesh API server
+    final var meshApiServer = injector.getInstance(MeshApiServer.class);
+    meshApiServer.start();
```

Comment:
(If we make the mesh API optional, then we will want to condition this start-up on either checking that it's enabled, or on checking `injector.getExistingBinding(Key.get(MeshApiServer.class)) != null`)